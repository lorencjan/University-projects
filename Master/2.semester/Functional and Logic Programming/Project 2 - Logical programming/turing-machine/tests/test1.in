S a S R
S b S R
S   F  
aaaaaabbaaaabbbbbbbbbb
