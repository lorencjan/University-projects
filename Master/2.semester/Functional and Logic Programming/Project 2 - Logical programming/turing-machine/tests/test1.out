Saaaaaabbaaaabbbbbbbbbb
aSaaaaabbaaaabbbbbbbbbb
aaSaaaabbaaaabbbbbbbbbb
aaaSaaabbaaaabbbbbbbbbb
aaaaSaabbaaaabbbbbbbbbb
aaaaaSabbaaaabbbbbbbbbb
aaaaaaSbbaaaabbbbbbbbbb
aaaaaabSbaaaabbbbbbbbbb
aaaaaabbSaaaabbbbbbbbbb
aaaaaabbaSaaabbbbbbbbbb
aaaaaabbaaSaabbbbbbbbbb
aaaaaabbaaaSabbbbbbbbbb
aaaaaabbaaaaSbbbbbbbbbb
aaaaaabbaaaabSbbbbbbbbb
aaaaaabbaaaabbSbbbbbbbb
aaaaaabbaaaabbbSbbbbbbb
aaaaaabbaaaabbbbSbbbbbb
aaaaaabbaaaabbbbbSbbbbb
aaaaaabbaaaabbbbbbSbbbb
aaaaaabbaaaabbbbbbbSbbb
aaaaaabbaaaabbbbbbbbSbb
aaaaaabbaaaabbbbbbbbbSb
aaaaaabbaaaabbbbbbbbbbS
aaaaaabbaaaabbbbbbbbbbF 
